backends/tests/e2e/integration_test.go

```diff
@@ -0,0 +1,79 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	authenticationv1 "k8s.io/api/authentication/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+	"knative.dev/reconciler-test/pkg/environment"
+	"knative.dev/reconciler-test/pkg/eventshub"
+	"knative.dev/reconciler-test/pkg/eventshub/assert"
+	"knative.dev/reconciler-test/pkg/feature"
+)
+
+var global environment.GlobalEnvironment
+
+func TestMain(m *testing.M) {
+	global = environment.NewStandardGlobalEnvironment()
+
+	// Run the tests.
+	os.Exit(m.Run())
+}
+
+func TestIntegration(t *testing.T) {
+	t.Parallel()
+
+	ctx, env := global.Environment(environment.Managed(t))
+	env.Test(ctx, t, TesAuth())
+}
+
+func TesAuth() *feature.Feature {
+	f := feature.NewFeature()
+
+	authenticatedClientName := feature.MakeRandomK8sName("authenticated-client")
+	unauthenticatedClientName := feature.MakeRandomK8sName("unauthenticated-client")
+
+	SAName := "sa"
+	SANamespace := "default"
+
+	f.Setup("request with authenticated client", func(ctx context.Context, t feature.T) {
+		tr := &authenticationv1.TokenRequest{
+			// TODO: fill up
+		}
```

Comment:
For this one, I would create a ServiceAccount (SA) with the expected permissions (ClusterRole and ClusterRoleBinding) in `test/config/` and apply it as part of the `knative_setup` (see my top-level review comment) and then this TokenRequest will use that SA to get an associated token and pass it to eventshub.